modules/controlfreec/1.1/controlfreec.smk

```diff
@@ -78,7 +65,6 @@ chromRegions = ("1", "2", "3", "4", "5", "6", "7", "8",
         "17", "18", "19", "20", "21", "22", "X", "Y")
```

Comment:
I'm not sure how these are getting "chr" added for hg19 or hg38. 

There's a file in the reference file workflow `genomes/{genome_build}/genome_fasta/main_chromosomes.txt`. You could check out this [checkpoint rule](https://github.com/LCR-BCCRC/lcr-modules/blob/5569794beb005874a67e767fb970f9b8eda3b5ab/modules/mutect2/2.0/mutect2.smk#L51) and [this function](https://github.com/LCR-BCCRC/lcr-modules/blob/5569794beb005874a67e767fb970f9b8eda3b5ab/modules/mutect2/2.0/mutect2.smk#L153) from the mutect module for how to use that for parallelizing by chromosome.

Reply:
The chromRegions variable is only used in the final _check_chrFiles rule, which generates a touch(.output.done) file to make sure all individual {chrom}.fa files have been accounted for. It has no bearing on the contents of the files.

My _generate_chrFasta rule is just my quick and dirty way to grep each chromosome (with or without "chr") depending on the genome_build

Reply:
It's definitely preferable to use the existing chromosome name files rather than hard-coding anything